pytmx/pytmx.py

```diff
@@ -1382,14 +1384,14 @@ def _set_properties(self, node):
         self.height = int(self.height)
         self.width = int(self.width)
 
-    def parse_xml(self, node: ElementTree.Element):
-        """Parse a Tile Layer from ElementTree xml node.
+    def parse_xml(self, node: ElementTree.Element) -> "TiledTileLayer":
+        """Parse a tile Layer from ElementTree xml node.
```

Comment:
i think that this should remain "Tile Layer", since that is the [proper] name of the thing being parsed

Reply:
alternatively, lowercase both words: "tile layer"